### ISSUE TITLE
Execution Failed: Mainfest merger failed how to solve

### ISSUE THREAD

**Issue by dipnaze32** [pedant/sweet-alert-dialog#128]
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@label value=(SociofeedbackV2) from AndroidManifest.xml:22:9-40
  	is also present at [com.pnikosis:materialish-progress:1.0] AndroidManifest.xml:14:9-41 value=(@string/app_name).
  	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:19:5-94:19 to override.

**Comment by adamis** [comment]
Got it working?

**Comment by F0RIS** [comment]
you shoud add in your manifest to application tag
```tools:replace="android:label"```
https://github.com/F0RIS/sweet-alert-dialog

**Comment by dipnaze32** [comment]
Yes Thanks guys. I added the tools: and change the varies style.

**Comment by ibadurrahmantalha** [comment]
add to your AndroidManifest.xml
into <application
tools:replace="android:icon"

and also in
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
usethis line ===>>> xmlns:tools="http://schemas.android.com/tools"
after adding clean your app and install it will solve your issue.

**Comment by zulkarnain-shah** [comment]
Still the same error. I am writing app in **Kotlin**
